KubeArmor/build/compile.sh

```diff
@@ -11,4 +11,13 @@ else
     make
 fi
 
-cp *.bpf.o ignore.lst /opt/kubearmor/BPF/
\ No newline at end of file
+cp *.bpf.o ignore.lst /opt/kubearmor/BPF/
+
+# checking and mounting bpffs if it isn't
+if mountpoint -q /sys/fs/bpf;
+then
+	echo "bpffs already mounted";
+else
+	 echo "bpffs not mounted. Mounting now.";
+	 mount -t bpf bpffs /sys/fs/bpf
+fi
```

Comment:
Can we also add the check in the main container? We might be eliminating init container in the future.

Reply:
Gotcha :+1:

Reply:
But the caveat mentioned in the other comment. We'll need to make it privileged.